net-im/psi/psi-9999.ebuild

```diff
@@ -67,6 +66,7 @@ DEPEND="${RDEPEND}
 PDEPEND="
 	crypt? ( app-crypt/qca[gpg] )
 	ssl? ( app-crypt/qca:2[ssl] )
+	webp? ( dev-qt/qtimageformats )
```

Comment:
this looks strictly runtime only, right? I'm (and QA is) usually against that, as long as portage does not properly support it (meaning, no rebuild necessary as the install image is not changed). There are otherwise two options, make it unconditional (this is a fairly light dependency) or put an info into pkg_postinst about the option.

Reply:
Then I'll make unconditional app-crypt/qca:2[ssl] and dev-qt/qtimageformats and add app-crypt/qca[gpg] to postinst